.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy Docs
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  rustdoc:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: hecrj/setup-rust-action@master
+      - uses: actions/checkout@master
+      - name: Build docs
+        run: |
+          cargo doc --all-features --no-deps
```

Comment:
While openmls is still under heavy development, it could be interesting to use

`--document-private-items`

as well.

Reply:
Yeah, but I wouldn't want it on the same page. Maybe let's set up two different directories. One with private and with the public API only.

Reply:
Sure, that works too.

Reply:
ok, not much difference here right now and not worth the additional effort. I added private functions.